modules/Configuration.py

```diff
@@ -0,0 +1,73 @@
+# coding=utf-8
+from ConfigParser import SafeConfigParser
+
+config = SafeConfigParser()
+# This module is the middleman between the bot and a SafeConfigParser object, so that we can add extra functionality
+# without clogging up lendingbot.py with all the config logic. For example, added a default value to get().
+
+
+def init(file_location):
+    loaded_files = config.read(file_location)
+    if len(loaded_files) != 1:
+        import shutil
+        # Copy default config file if not found
+        shutil.copy('default.cfg.example', 'default.cfg')
+        print '\ndefault.cfg.example has been copied to default.cfg\n' \
+              'Edit it with your API key and custom settings.\n'
+        raw_input("Press Enter to acknowledge and exit...")
+        exit(1)
+    return config
+
+
+def has_option(category, option):
+    return config.has_option(category, option)
+
+
+def getboolean(category, option):
+    return config.getboolean(category, option)
+
+
+def get(category, option, default_value=False, lower_limit=False, upper_limit=False):
```

Comment:
I mentioned that usage in a comment on lendingbot.py, but I definitely intend to update the  docs with this iinformation.